README.md

```diff
@@ -17,14 +17,25 @@ using (var httpClient = new HttpClient())
      var sendDocument = await citizenDocumentClient.SendDocumentWithHttpMessagesAsync(new Guid(configuration.SubscriptionId), new SendCitizenDocumentRequest
 }
 ```
+The simplest example for companyDocuments where you can upload or send documents:
+ 
+```csharp
+using (var httpClient = new HttpClient())
+{
+     var tokenProviderFactory = new LogicTokenProviderFactory(tokenProviderOptions);
+     var companyDocumentClient = new CompanyDocumentsClient(httpClient, tokenProviderFactory, configuration.Citizen);
+     var uploadDocument = await companyDocumentClient.UploadAttachmentWithHttpMessagesAsync(new Guid(configuration.SubscriptionId), configuration.ConfiguartionId, configuration.RetentionPeriodInDays, configuration.Cpr, configuration.DocumentType, configuration.Document, configuration.DocumentName).ConfigureAwait(false);
+     var sendDocument = await companyDocumentClient.SendDocumentWithHttpMessagesAsync(new Guid(configuration.SubscriptionId), new SendCitizenDocumentRequest
+}
+```
 
 The `LogicTokenProviderFactory` authorizes access to the Logic platform through the use of a Logic Identity issued client credential. The authorization token is reused until it  expires. You would generally create a single instance of `LogicTokenProviderFactory`.
 
-The `CitizenDocumentsClient` accesses the Logic CitizenDocuments service which in turn interacts with one of the data providers.
+The `CitizenDocumentsClient` accesses the Logic CitizenDocuments and `CompanyDocumentsClient` accesses the Logic CompanyDocuments service which in turn interacts with one of the data providers.
```

Comment:
currently we don't have any explicit thing rite naming company documents ?
Bit doubtful here